build.sbt

```diff
@@ -1,82 +1,91 @@
+import com.softwaremill.Publish.ossPublishSettings
```

Comment:
would also be good to add CI + publishing as in our other projets, see e.g. sttp-model for a simple example on how to configure github actions

Reply:
Okay